src/configs/env.ts

```diff
@@ -8,10 +9,12 @@ type EnvConfig = {
   nodeEnv: 'development' | 'test' | 'production';
   port: string;
   logs: 'common' | 'dev';
+  domain: string;
 };
 
 export default {
   nodeEnv,
   port: process.env.PORT || '8080',
   logs: 'dev',
+  domain: `http://${process.env.DB_HOST}:${process.env.PORT}`,
```

Comment:
Don't use DB_HOST for domain, you should create new one

Reply:
+